scripts/lib/mail_credentials_helper.rb

```diff
@@ -1,3 +1,5 @@
+`echo #{File.expand_path(__FILE__, __dir__)} >> ~/.paas-script-usage`
```

Comment:
looks like this is the last one that shells out

Reply:
Bleg! Find and replace has betrayed me! I'll fix it, once and for all ⚔️